src/SelectField/SelectField.jsx

```diff
@@ -216,6 +224,7 @@ const SelectField = React.createClass({
           style={selectFieldRoot}
           labelStyle={Object.assign(styles.label, labelStyle)}
           iconStyle={Object.assign(styles.icon, iconStyle)}
+          immediatelyOpen={autoFocus}
```

Comment:
If you have the source, you have a running test env! (The docs site can be used of manual testing, there is a also a new test suite, but coverage isn't there yet.)